hmpps-prisoner-search-indexer/src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/indexer/services/dto/nomis/OffenderBooking.kt

```diff
@@ -1,13 +1,16 @@
 package uk.gov.justice.digital.hmpps.prisonersearch.indexer.services.dto.nomis
 
+import org.apache.commons.lang3.builder.DiffBuilder
+import org.apache.commons.lang3.builder.DiffResult
+import org.apache.commons.lang3.builder.Diffable
+import org.apache.commons.lang3.builder.ToStringStyle
 import java.time.LocalDate
 
 data class OffenderBooking(
   val offenderNo: String,
   val firstName: String,
   val lastName: String,
   val dateOfBirth: LocalDate,
-  val activeFlag: Boolean,
```

Comment:
Accidentally copied a bunch of properties that aren't returned from the new endpoint - and weren't used even though returned from the old endpoint.